t/config-center-yaml/global-rule.t

```diff
@@ -112,3 +112,18 @@ global_rules:
                     - /h*
 #END
 --- error_code: 403
+
+
+
+=== TEST 4: common phase without matched route
+--- apisix_yaml
+global_rules:
+    -
+        id: 1
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/105040972-6c673300-5a9d-11eb-8c6e-801ce6fd7129.png)

Reply:
The indentation is correct. The yaml can be parsed correctly.